providers/airbyte/tests/conftest.py

```diff
@@ -16,17 +16,4 @@
 # under the License.
 from __future__ import annotations
 
-import pathlib
```

Comment:
All other `conftest.py` removals in 90+ providers follow the same pattern. We are using warning deprecation impleented in the `pytest_plugin`